static/app/components/forms/fields/segmentedRadioField.tsx

```diff
@@ -0,0 +1,179 @@
+import isPropValid from '@emotion/is-prop-valid';
+import styled from '@emotion/styled';
+
+import type {RadioGroupProps} from 'sentry/components/forms/controls/radioGroup';
+import type {InputFieldProps, OnEvent} from 'sentry/components/forms/fields/inputField';
+import FormField from 'sentry/components/forms/formField';
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import Radio from 'sentry/components/radio';
+import {Tooltip} from 'sentry/components/tooltip';
+import {space} from 'sentry/styles/space';
+
+export interface SegmentedRadioFieldProps extends Omit<InputFieldProps, 'type'> {
+  choices?: RadioGroupProps<any>['choices'];
```

Comment:
🤔 should this `any` propagate the generic type down instead?

Reply:
Ah great catch, missed this in the copy + paste. Fixed in 9541fb87697!